Cork/Views/Settings/Panes/General Pane.swift

```diff
@@ -23,19 +23,19 @@ struct GeneralPane: View
             {
                 Picker(selection: $sortPackagesBy)
                 {
-                    Text("Alphabetically")
+                    Text("Name")
                         .tag(PackageSortingOptions.alphabetically)
-                    Text("By Installation Date")
+                    Text("Installation Date")
                         .tag(PackageSortingOptions.byInstallDate)
-                    Text("By Size")
+                    Text("Size")
                         .tag(PackageSortingOptions.bySize)
                     
                     Divider()
                     
                     Text("Do Not Sort")
                         .tag(PackageSortingOptions.none)
                 } label: {
-                    Text("Sort packages:")
+                    Text("Sort packages by:")
```

Comment:
Finder's 'View Options' window (right-click inside a folder) also uses 'Sort by:' and 'Group by:'

Reply:
I agree with this